module3/assessments/week1-9715.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: page
+title: Week 3 Assessment
+---
+
+[Follow the instructions on this repo!](https://github.com/turingschool-examples/Launch_Mod3Wk1_Assessment)
```

Comment:
Yay already making it a separate repo so we don't have to do that the week before!!

The link is not working though, I think this is the correct link: https://github.com/turingschool-examples/Launch_Mod3Week1_Assessment

Reply:
How do we want them to turn in the repl? Maybe they add the link into their GitHub repo somewhere?

Reply:
haha yep - I made the link before I made the repo 😆

Reply:
The question mark is in a weird place in question 4, which was totally my doing 😆 How do you describe the difference between HTML and CSS, use an analogy or an example in your answer?

Reply:
Just noticed it still says Week 3 Assessment in the assessment Repo

Reply:
updated for these last two comments!